editor/src/components/canvas/canvas-strategies/strategies/reparent-helpers/reparent-property-changes.ts

```diff
@@ -90,6 +90,7 @@ export function getAbsoluteReparentPropertyChanges(
   newParentStartingMetadata: ElementInstanceMetadataMap,
   projectContents: ProjectContentTreeRoot,
   forcePins: ForcePins,
+  containLayout: 'should-add' | 'should-not-add',
```

Comment:
see my earlier comment, I would also rename this var to `shouldAddContainLayout` or something like that

Reply:
Parameter renamed in 788e1b602.